extensions/sharing/pkg/config/config.go

```diff
@@ -108,7 +108,7 @@ type PublicSharingDrivers struct {
 }
 
 type PublicSharingJSONDriver struct {
-	File string `yaml:"file" env:"SHARING_PUBLIC_JSON_FILE"`
+	File string `yaml:"file" env:"SHARING_PUBLIC_JSON_FILE" desc:"Path to the json file where public shares will be persisted."`
```

Comment:
Sense is not clear, needs rewording.

Reply:
"Path to the json file where public share meta-data will be stored"

I'm not sure what exactly to write that is different. There is a JSON file that stores the information about public share endpoints that have been created. The actually data is files/folders/resources that exist somewhere in a space (personal and/or project...)

Reply:
A good sentences you posted describing what it is 👍 just summarizing as discussion to be added as commitable suggestion in case you agree:
`Path to the JSON file where public share meta-data will be stored. This JSON file contains the information about public share endpoints that have been created.`

What is the default?

Reply:
The default is defined in https://github.com/owncloud/ocis/blob/ba9a48a510b20bdae605141d395f95d7c8ab7539/extensions/sharing/pkg/config/defaults/defaultconfig.go#L55. The default value will be available in the generated documentation.